### PR TITLE
Modulesync 0.19.3 and more

### LINKED ISSUE #39: Travis shows acceptance tests green but are not run
travis  tries to run `bundle exec rake $CHECK`($CHECK not substituted) which lists just all the tasks and exits 0

did try to run them on my machine. at least the docker nodesets need an update because `ss` is not in the install list and is required. 

spec/spec_helper_acceptance.rb also requires an update.

### PR DESCRIPTION
**Pull request by vinzent**
Seems tests are failing since 1 year.

Unfortunatly I needed to skip some rspec tests. they work on my computer but not on travis. no idea what cold be wrong. someone with more ruby/augeas knowledge needs to have a look. I opend issue #41 

Closes #39

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -519,5 +519,26 @@ RSpec/RepeatedDescription:
...
-  Enabled: False
+  Enabled: false
+
+
+#------------------------------
+# CUSTOM 
+
+# Offense count: 2
+# Configuration parameters: AssignmentOnly.
+RSpec/InstanceVariable:
```
Comment (bastelfreak) [inline .rubocop.yml]: this whole block will be overwritten by the next modulesync

Comment (bastelfreak) [inline .rubocop.yml]: maybe disable the cops inside of the spec file?

Comment (vinzent) [inline .rubocop.yml]: there is more overriden next modulesync. all in https://github.com/voxpupuli/puppet-puppetserver/pull/40/commits/f711d505a9b5c51e2ecfcfefd15f63ed037c8b76

if you know a better way just fix it yourself. :-)

**Comment by bastelfreak** [comment]
thanks \o/

**Pull request merged**